hack/license-header-check.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright the KubeVirt Authors.
+
+set -euo pipefail
+
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)"
+
+TARGET_DIRS=(
+    "pkg/monitoring"
+)
+
+IGNORE_FILES=(
+    # Example: "pkg/monitoring/ignore_this_file.go"
+)
+
+MISSING_LICENSE_FILES=()
+
+# Define the required license lines (order does not matter)
+LICENSE_LINES=(
```

Comment:
Lets parse the license lines from `hack/boilerplate` for the sake of consistency. Otherwise, being inconsistent, in the future if we will apply the linter on other sections, we may hit lint failure on the auto generated go code because whats in the hack/boilerplate is different from what is in the script.

Reply:
sure but it needs to be updated to the correct template we use for most files.